'npm run bootstrap' fails on Linux (Mint 18.3)

I get the following when executing `npm run bootstrap`. 

I suspect the issue is your scripts cal out `/bin/sh` but on Mint the default shell is **dash** and apparently this script isn't _dash_ compatible. 

When I forced this script to execute with **bash** it succeeded.

```console
~/s/material-components-web-components $ npm run bootstrap

> material-components-web-components@ bootstrap /home/shawn/s/material-components-web-components
> npm install && ./scripts/bootstrap-unreleased.sh

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

removed 97 packages in 12.261s
lerna info version 2.11.0
lerna info Bootstrapping 23 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna info hoist Finished installing in root
lerna info Symlinking packages and binaries
lerna info lifecycle postinstall
lerna info lifecycle prepublish
lerna info lifecycle prepare
lerna success Bootstrapped 23 packages
./scripts/bootstrap-unreleased.sh: 24: ./scripts/bootstrap-unreleased.sh: Syntax error: "(" unexpected
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! material-components-web-components@ bootstrap: `npm install && ./scripts/bootstrap-unreleased.sh`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the material-components-web-components@ bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/shawn/.npm/_logs/2018-05-06T15_04_44_169Z-debug.log
```

--- a/scripts/bootstrap-unreleased.sh
+++ b/scripts/bootstrap-unreleased.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 ##
 # Copyright 2016 Google Inc. All Rights Reserved.

--- a/scripts/update-styling.sh
+++ b/scripts/update-styling.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 ##
 # Copyright 2016 Google Inc. All Rights Reserved.

